test/robot_test.coffee

```diff
@@ -35,6 +35,7 @@ describe 'Robot', ->
 
     # Re-throw AssertionErrors for clearer test failures
     @robot.on 'error', (name, err, response) ->
+      return unless err?
```

Comment:
I managed to create some situations while testing that had an undefined `err` here. I'm not quite sure what I did, but the re-raiser was raising its own errors and it wasn't helpful :grin: